### ISSUE TITLE
The `tweets` folder is not recognized.

### ISSUE THREAD

**Issue by ThreeSnakes** [gr2m/twitter-together#81]
Hello. 

I am applying the **twitter-together** you made to my repertoire.

But I think the application went well, but there is one problem.

![스크린샷 2020-01-03 오후 12 19 57](https://user-images.githubusercontent.com/36795031/71705489-5f8c8880-2e23-11ea-9a62-238be729bc7c.png)

The action does not seem to recognize the **tweets** directory. Did I do anything wrong?

[repo Link](https://github.com/ThreeSnakes/TIL)

Thank you.

**Comment by ThreeSnakes** [comment]
I think I found a solution.

I output the `GITHUB_WORKSPACE` variable from the current code to the console, the structure of that directory will look at ls -al and it will look like this:
![스크린샷 2020-01-03 오후 2 28 07](https://user-images.githubusercontent.com/36795031/71708511-4d1b4a80-2e35-11ea-8ebe-18fe7fbd0005.png)

As in the image above, you can see that the directory is empty.

So I looked for a way, and I found this [link](https://github.community/t5/GitHub-Actions/Github-Actions-workspace-is-empty/td-p/29081).

As shown in the document, add `- us: actions/checkout@v1` to the step, and then retake the log.

<img width="581" alt="스크린샷 2020-01-03 오후 2 41 27" src="https://user-images.githubusercontent.com/36795031/71709374-0da32d00-2e3a-11ea-9928-652c6cadfc1b.png">

As shown in the picture above, I read the story well and found the twiets directory well and succeeded in sending the tweet.

The complete log can be viewed at the following link: [LINK](https://github.com/ThreeSnakes/TIL/commit/66b2af639db93f5f94a40fb196f791edd266d317/checks?check_suite_id=382564761)

**Comment by ThreeSnakes** [comment]
I'll also copy the yml file used for the test.

``` yml
on: [push, pull_request]
name: Twitter, together!
jobs:
  preview:
    name: Preview
    runs-on: ubuntu-latest
    if: github.event_name == 'pull_request'
    steps:
      - name: Preview
        uses: gr2m/twitter-together@master
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  tweet:
    name: Tweet
    runs-on: ubuntu-latest
    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
    steps:
      - name: console 1
        run: |
          echo ${{ github.workspace }}
          ls -al
      - name: use action/checkout
        uses: actions/checkout@v1
      - name: console 2
        run: |
          echo ${{ github.workspace }}
          ls -al
      - name: Tweet
        uses: gr2m/twitter-together@master
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          TWITTER_ACCESS_TOKEN: ${{ secrets.TWITTER_ACCESS_TOKEN }}
          TWITTER_ACCESS_TOKEN_SECRET: ${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
          TWITTER_API_KEY: ${{ secrets.TWITTER_API_KEY }}
          TWITTER_API_SECRET_KEY: ${{ secrets.TWITTER_API_SECRET_KEY }}
```

**Comment by gr2m** [comment]
Oh I see, the `uses: actions/checkout@v1` action is needed before the tweet action? 

You should use `gr2m/twitter-together@v1.x` though, or did that not work for you?

**Comment by ThreeSnakes** [comment]
I tested it again because I might have done something wrong.

use `gr2m/twitter-together@v1.x` and remove `uses: actions/checkout@v1` from `.yml file`

The result came out like the next screen shot.

![스크린샷 2020-01-04 오전 8 40 26](https://user-images.githubusercontent.com/36795031/71755550-72a86280-2ece-11ea-9cc0-9f2798efc1ea.png)

if `uses: actions/checkout@v1` is removed from the `.yml file`, `Tweet action` still cannot find the `tweets folder`.

So this time, I tested it with `uses: actions/checkout@v1` added to the `.yml file` and use `gr2m/twitter-together@v1.x`.

![스크린샷 2020-01-04 오전 8 49 49](https://user-images.githubusercontent.com/36795031/71755727-6670d500-2ecf-11ea-886f-054b988543f5.png)

The `Tweet action` recognizes the `tweets folder` well by adding `uses: actions/checkout@v1`.

RESULT: For action to recognize the `tweet folder`, `uses: actions/checkout@v1` should be put into the `.yml file` before `Tweet action`.

I am sorry that I am not good at English. 😢

**Comment by gr2m** [comment]
Your English is great, and your screenshots are very helpful. Thank you for all your help. Do you want to send a pull request to update the instructions?

You need to update two files

1. [README.md](https://github.com/gr2m/twitter-together/blob/master/README.md)
2. [docs/02-create-twitter-together-workflow.md](https://github.com/gr2m/twitter-together/blob/master/docs/02-create-twitter-together-workflow.md)

**Comment by ThreeSnakes** [comment]
Oh. Thank you for giving me a chance. I'll give it a try.

**Comment by gr2m** [comment]
that was resolved via #83
